ontology-modeling.markdown

```diff
@@ -68,14 +68,14 @@ Ontologies, and also SPARQL queries and N3-rules, are created with the text edit
 <!---iterative epistemological hermeneutic process (Pierazzo 2015)--->
 
 ### A Multitude of Networked Namespaces
-In creating a semantic space for broad applicability, also ontology size and consequently the number of ontologies developed matter. Having “all contained in one” or even a few very big ontologies is rather difficult if the envisioned research space to talk about is broad, implying multiple domains. A basic approach is a multitude of networked namespaces, a web of highly interdependent ontologies (Figure 5). They differ in size, granularity and specificity.
+In creating a semantic space for broad applicability also ontology size and consequently the number of ontologies developed matter. Having “all contained in one” or even a few very comprehensive ontologies is rather difficult if the envisioned research space is broad, implying multiple domains. A basic approach is a multitude of networked namespaces, a web of highly interdependent ontologies (Figure 5). They differ in size, granularity and specificity.
 
 {% include image.html type="intermediate-figure" url="/assets/images/ontologies-network.png" description="Figure 5: A simplified representation of the e-editiones web of ontologies" %}  
 
-A scientific project always needs general concepts such as ‘person’, ‘document’, text’, ‘image’, reusing elements from more general ontologies. A project ontology can exist, but should base on the general ones, and only contain project-specific elements, e.g. the concept of “Dreissiger” (set of about 30 verses) used in the Parzival-project.  
+A scientific project always needs general concepts such as ‘person’, ‘document’, text’ and ‘image’, reusing elements from more general ontologies. A project ontology can exist, but should base on the general ones and only contain project-specific elements, e.g. the concept of “Dreissiger” (set of about 30 verses) used in the Parzival-project.  
 
 ### Delineation and Naming
-One of the bigger challenges, emerging from previous topic is how to partition knowledge over different namespaces, in other words to choose a workable delineation and hence name along the content (domain of discours) of an ontology. Another basic approach in this procedure, enabled by this choice, is **easy addition and extension**, and **minimize the need for later splitting** of an ontology. On the other hand some domain knowledge gets outdated, other updated, but it is expected in Humanities for knowledge not to change as fast as in natural sciences, hence a lower risk for deprecation of ontological elements.  
```

Comment:
I don't really see the link to the last sentence. But it can be omitted without any problems, right?

Reply:
Last sentence rewritten:
'On the other hand some domain knowledge gets outdated, other updated, leading to deprecation of ontological elements.'